### ISSUE TITLE
Unreal Editor Log is Totally Empty

### ISSUE THREAD

**Issue by HISYD** [JetBrains/UnrealLink#262]
I'm a beginner in learning Unreal and C++. It puzzled me a that in Unreal Editor Log is totally empty and the information in Unreal Output Log doesnot show here as I wished. I have tried to uninstall rider or Plugin and install them again, but it did not help...

**Comment by hellg1** [comment]
Hello @HISYD!
Do you have RiderLink plugin installed in your game project? Does it happen only in one project or every project is affected?

**Comment by HISYD** [comment]
Oh actually I have tried installing it in Game or Engine, neither of them worked...I tried to create a pure project based on 4.27 (epic launcher version), it failed to show anything in Unreal Editor Log. This action was repeated also in source code build 5.2 version and the result is same...

ps. sorry i'm not a native English speaker, forgive my poor English....lol
Thank you for your reply!

**Comment by nameless-user12138** [comment]
You mean the Unreal Editor Log tab is showed but there is no logging? Better post a picture.

**Comment by Dutchi-Who** [comment]
Yes I met the same issue. I cannot find the tab `Unreal Log` in `View->Tool Windows`.

![image](https://github.com/JetBrains/UnrealLink/assets/96360144/94b64c6f-7c0a-4c0e-a857-62a3b5f8bf8d)

I've tried a solution in [other comments of this problem](https://youtrack.jetbrains.com/issue/RIDER-68441/Unreal-Editor-Log-Disappeared#focus=Comments-27-6892108.0-0), which is disable and re-enable the `UnrealLink` plugin, but it didn't work for me. I saw this plugin no longer has major updates and the version stopped at year 2020.

So does anyone meet the same issue in Rider version same or later than 2023?

**Comment by DecoyRS** [comment]
Hello folks!
In order to show logs in Unreal Log panel, next requirements should be in place:
* UnrealLink plugin is enabled inside Rider
* Compatible RiderLink plugin is installed in the Editor or Game. RiderLink is bundled with UnrealLink inside Rider and you should have a notification if versions don't match each other
* Unreal Editor should be running. In that case UnrealLink and RiderLink will establish connection and show you the logs.

@Dutchi-Who UnrealLink is updated with every version of Rider. In extreme cases like fixing crashes we might release an intermediate version on Marketplace, but it's been awhile since we posted one on the Marketplace.

Can you post your current version of UnrealLink and RiderLink installed?

**Comment by Dutchi-Who** [comment]
Here's my Unreal and Rider plugins' versions:
- Unreal 5.3.2 (from Epic Games Store)
- Rider 2023.2.3 (from JetBrains Toolbox)
- RiderLink 2023.2.0.213 (installed in Engine)
- UnrealLink 2023.2.0.213 (bundled)

![image](https://github.com/JetBrains/UnrealLink/assets/96360144/473a9410-2d09-4667-a546-c0bf97d56ef4)
![image](https://github.com/JetBrains/UnrealLink/assets/96360144/9623571f-bfd1-4141-a9f8-2614fbf02542)
![image](https://github.com/JetBrains/UnrealLink/assets/96360144/c02c5fc3-23bb-41bc-8b7f-eb8abb611a07)
![image](https://github.com/JetBrains/UnrealLink/assets/96360144/aae7a7d2-bd25-429c-acfb-97a6262f548d)

Meanwhile, when I tried to navigate to Blueprint in C++, it popped this warning. I didn't find any error message except this. 

![image](https://github.com/JetBrains/UnrealLink/assets/96360144/0b3200c5-d9d3-443b-ac22-5ec71c423ca3)

Do you think that would help? I met all requirements but the `Unreal Log` tab just doesn't appear.

**Comment by andreiagmu** [comment]
I'm not sure if this may be related to your issue, but today, I had an issue with RiderLink:

Every time I opened my Rider project, it always asked me to install RiderLink.
And every time I tried to install it, using either `Install plugin in Engine` or `Install plugin in Game` options, nothing happened.
Thus, my Rider project was never connecting to Unreal Engine, always showing `Not Connected to Unreal Editor` status.

Turns out I needed to open Rider with Admin privileges.
In JetBrains Toolbox, I used the `Run as administrator` option (in the "three dots" menu at the right of Rider), and then I was finally able to install RiderLink!

Now, Rider connects to the Unreal Editor, and the UnrealLink (Unreal Editor Log) view appears correctly in Rider. :D

**Comment by Dutchi-Who** [comment]
> I'm not sure if this may be related to your issue, but today, I had an issue with RiderLink:
> 
> Every time I opened my Rider project, it always asked me to install RiderLink. And every time I tried to install it, using either `Install plugin in Engine` or `Install plugin in Game` options, nothing happened. Thus, my Rider project was never connecting to Unreal Engine, always showing `Not Connected to Unreal Editor` status.
> 
> Turns out I needed to open Rider with Admin privileges. In JetBrains Toolbox, I used the `Run as administrator` option (in the "three dots" menu at the right of Rider), and then I was finally able to install RiderLink!
> 
> Now, Rider connects to the Unreal Editor, and the UnrealLink (Unreal Editor Log) view appears correctly in Rider. :D

I tried, thanks for your advice, but admin privilege cannot help on my computer :-(

**Comment by kapkrash** [comment]
![image](https://github.com/JetBrains/UnrealLink/assets/105563943/5550f381-6fd7-4ba5-a521-654d114205a2)
![image](https://github.com/JetBrains/UnrealLink/assets/105563943/2c5074d8-87dc-4e85-8f55-7acb7c5d6814) 
![image](https://github.com/JetBrains/UnrealLink/assets/105563943/5481d6e6-1c82-4ff2-95ab-b46ae1be9dda)
![image](https://github.com/JetBrains/UnrealLink/assets/105563943/47c000c8-10c9-4af7-bb00-29e6eeb0e593)
![image](https://github.com/JetBrains/UnrealLink/assets/105563943/7c535870-e76b-490f-bbae-a61abf228ec7)
![image](https://github.com/JetBrains/UnrealLink/assets/105563943/93c39477-57d5-4e42-8d36-262b91025e03)


So I have all the settings correct. I also have admin rights on it. I have no idea why its not working :(
